Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash
```

Comment:
Seems unrelated.

Reply:
it is necessary since make test calls a bash script. without it Makefile uses /bin/sh by default and the lane fails on some bash specific commands

Reply:
this commit moved to https://github.com/metallb/metallb-operator/pull/43, same applies there